components/automate-ui/src/app/modules/infra-proxy/select-box/src/lib/select-box.component.html

```diff
@@ -6,7 +6,7 @@
                 <div class="input-icons role-list-margin">
                     <mat-icon class="icon">search</mat-icon>
                     <input chefInput *ngIf="!disabled && search" type="text" class="selectbox-search input-field"
-                    [(ngModel)]="leftFilterText" [disabled]="disabled" placeholder="Search.." />
+                    [(ngModel)]="leftFilterText" [disabled]="disabled" placeholder="Search.." /> <!-- TODO: do not use two-way binding! -->
```

Comment:
A key tenet of automate design is One-Way Binding. The `[(ngModel)]` uses two-way binding and needs fixing; just have not done it here.

Reply:
Agreed, will fix these changes in #4540.